Gemfile.lock

```diff
@@ -55,52 +55,52 @@ GEM
       ransack (~> 2.1, >= 2.1.1)
       sassc-rails (~> 2.1)
       sprockets (>= 3.0, < 4.1)
-    activejob (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    activejob (6.0.3.1)
+      activesupport (= 6.0.3.1)
       globalid (>= 0.3.6)
-    activemodel (6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activerecord (6.0.2.2)
-      activemodel (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activestorage (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
+    activemodel (6.0.3.1)
+      activesupport (= 6.0.3.1)
+    activerecord (6.0.3.1)
+      activemodel (= 6.0.3.1)
+      activesupport (= 6.0.3.1)
+    activestorage (6.0.3.1)
+      actionpack (= 6.0.3.1)
+      activejob (= 6.0.3.1)
+      activerecord (= 6.0.3.1)
       marcel (~> 0.3.1)
-    activesupport (6.0.2.2)
+    activesupport (6.0.3.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.2)
+      zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
     annotate (3.1.1)
       activerecord (>= 3.2, < 7.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.5.17)
+    apipie-rails (0.5.18)
```

Comment:
### [apipie-rails](http://github.com/Apipie/apipie-rails)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.5.17 → 0.5.18

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **3** commits. See the full changes on [the compare page](https://github.com/Apipie/apipie-rails/compare/v0.5.17...v0.5.18).

These are the individual commits:
- (dfe5ef9) [Clean up Gemfile.rails60 and test on Ruby 2.7](https://github.com/Apipie/apipie-rails/commit/dfe5ef9c07e9e1cbb10ef1e0b31966b9f17e738e)
- (7ab1530) [Fixes #544: Optional rdoc dependency (#683)](https://github.com/Apipie/apipie-rails/commit/7ab1530d594fb08a406e1bd67de37cceaaa6a9b9)
- (cdff4ec) [Bump version to 0.5.17](https://github.com/Apipie/apipie-rails/commit/cdff4ec654ee70dd222c7d75f64e20b167b44fce)


</details>